install.sh

```diff
@@ -88,6 +85,96 @@ execute_installation() {
         setup_shared_hook_repositories
         echo # For visual separation
     fi
+
+    thank_you
+}
+
+############################################################
+# Apply legacy fixes if appropriate.
+#
+# Returns:  0
+############################################################
+apply_legacy_fixes() {
+
+    # if not installed this will be empty
+    INSTALLED_VERSION=$(git hooks version 2>/dev/null | grep "Version:" | sed -E "s/^Version: //")
+
+    # Updating from this version needs a single install fixture
+    LEGACY_FIXTURE_SINGLE_INSTALL=$(echo "$INSTALLED_VERSION 2004.272234-8af36c" | awk "{ print (\$1 <= \$2) }")
+
+    if [ "$LEGACY_FIXTURE_SINGLE_INSTALL" = "1" ] &&
+        LEGACY_FIXTURE_SINGLE_INSTALL="true"; then
+        apply_legacy_fix_single_install
+    fi
+
+    return 0
+}
+
+############################################################
+# Apply single install fixes for deperecated single installs
+#
+# Returns:  0
+############################################################
+apply_legacy_fix_single_install() {
+    if [ -f "$INSTALL_DIR/autoupdate/registered" ]; then
+
+        echo "Apply single install fixture to non-single installs ... "
+
+        LIST="$INSTALL_DIR/autoupdate/registered"
+        IFS="$IFS_NEWLINE"
+        while read -r INSTALLED_REPO; do
+            unset IFS
+
+            if ! is_git_repo "$INSTALLED_REPO"; then
+                # Not existing git dir -> skip.
+                true
+
+            else
+                echo "Migrate githooks install in $INSTALLED_REPO"
+                install_hooks_into_repo "$INSTALLED_REPO"
+            fi
+
+            IFS="$IFS_NEWLINE"
+        done <"$LIST"
+
+        # Delete the old list.
+        rm -rf "$LIST"
+
+        WHAT="single"
+    else
+        WHAT="single & non-single"
+    fi
+
+    echo "!! MIGRATION INFORMATION: From now on single installs are deprecated: "
```

Comment:
This says single installs are deprecated, but `--single` is still a supported option. Isn't the thing more that `--single` now no longer implies no autoupdate, but still installs just into a single repo (as opossed to searching the filesystem for all repos)?

Reply:
Exaclty --single is now meaning "one install into one repo" nothing more. Which is supported by the cli.sh.

The meaning was twofold before: --single - install into 1 repo, and no updates!

Reply:
So then the message should not say "single installs are deprecated", I think?